components/core/header/Header.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <header class="w-full h-full">
+    <header class="w-full" :style="{ height: '64px' }">
```

Comment:
One of the conventions in this project is avoiding inline style in the template. Please replace it with a newly-created class and add the CSS code for it in the `style` block.

Reply:
https://github.com/pycontw/pycontw-frontend/pull/218/commits/1134d4e93beb0aeed8480dbb2ec357bafcc790d2